src/Neo.Extensions/Neo.Extensions.csproj

```diff
@@ -15,7 +15,7 @@
   <ItemGroup>
     <InternalsVisibleTo Include="Neo" />
     <InternalsVisibleTo Include="Neo.IO" />
-    <InternalsVisibleTo Include="$(AssemblyName).Tests" />
+    <InternalsVisibleTo Include="Neo.UnitTests" />
```

Comment:
Move the tests to a different project, `Neo.Extensions.Tests`

Reply:
I would have to add setup for the `node` and everything else that comes with it. I think.

Reply:
I'll do it in another PR.

Reply:
Have no sense for me create a new pr for this, if the source code is moved, the unit test also should me moved

Reply:
It will break `PART-2`,`PART-3`,`PART-4`,`PART-5`, and `PART-6`?

Reply:
> It will break `PART-2`,`PART-3`,`PART-4`,`PART-5`, and `PART-6`.

Otherwise we have unit test in the wrong place, I prefer to know that master is more important than `PART-X`

Reply:
Can i add in `Part-7`?